frontend/packages/dev-console/src/components/taskruns/list-page/TaskRunsRow.tsx

```diff
@@ -42,7 +43,9 @@ const TaskRunsRow: RowFunction<TaskRunKind> = ({ obj, index, key, style, ...prop
     <TableData className={tableColumnClasses[3]}>
       {obj.spec.taskRef?.name ? (
         <ResourceLink
-          kind={taskReference}
+          kind={
+            obj.spec.taskRef?.kind === ClusterTaskModel.kind ? clusterTaskReference : taskReference
```

Comment:
So we already have this kinda check ... but not quite this _exact_ check in the [pipeline-augment.ts](https://github.com/openshift/console/blob/master/frontend/packages/dev-console/src/utils/pipeline-augment.ts#L478-L490) file. Perhaps we just expand this with another wrapper function:

Ie, add this:
```
const getModelReferenceFromTaskKind = (kind: string): GroupVersionKind => {
  const model = getResourceModelFromTaskKind(kind);
  return referenceForModel(model);
}
```

Reply:
So sorry for the confusion... please place this utility with the others in the `pipeline-augment.ts` file noted above.